.cypress/integration/3_panels.spec.ts

```diff
@@ -18,24 +18,6 @@ import {
 
 import { suppressResizeObserverIssue } from '../utils/constants';
 
-const moveToEventsHome = () => {
```

Comment:
are we using cypress from https://github.com/opensearch-project/opensearch-dashboards-functional-test/ repo or here?